README.md

```diff
@@ -1 +1,170 @@
-# ruby-capstone
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Ruby capstone project ](#-ruby-capstone-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+    - [Usage ](#usage-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Ruby-Capstone <a name="about-project"></a>
+
+**Ruby Capstone** This app is designed with Ruby and allows a user to add books, music_albums, and movies.
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+- Ruby
+
+
+### Video Presentation 
+Kindly take a look at our video presentation [here](https://youtu.be/eJKHfmB4RB8) 
```

Comment:
- [ ] Nice and engaging presentation, please according to the project requirement; kindly ensure that the recorded video is not longer than 5mins and also highlight some interesting piece of code or something you built that you are very proud of. 👍

Reply:
A new video has been recorded.
thanks for the tip. 👌🏿